CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to this project will be documented in this file.
 - Add support for ```tags```, ```status``` and ```annotation``` in Zipkin exporter.
 - Add support for Binary propagation format.
 - Add support for object(```SpanOptions```) as an argument for ```startChildSpan``` function, similar to ```startRootSpan```.
+- Add proto files to exporter-ocagent package. Fixes issue #174. 
```

Comment:
Nit, should `issue #174` be a link? Or is that done automatically?

Reply:
Good point, likely not done automatically. I'll add as link.